Userland/Libraries/LibAudio/PlaybackStreamOboe.cpp

```diff
@@ -49,9 +49,9 @@ class OboeCallback : public oboe::AudioStreamDataCallback {
         : m_data_request_callback(move(data_request_callback))
     {
     }
-    Duration last_sample_time() const
+    AK::Duration last_sample_time() const
```

Comment:
Please explain in the commit message why the `Duration` -> `AK::Duration` change is needed, to document it for the future.

Reply:
Okay, I have added this to the commit:
AK::Duration isn't exported into global namespace since commit bf600c8
It basically does the same job as commit 6772d44